test/Microsoft.Dnx.DesignTimeHost.FunctionalTests/Infrastructure/DthFunctionalTestFixture.cs

```diff
@@ -52,17 +48,35 @@ public IDisposable CreateDisposableTestProject(string projectName, string runtim
 
         private void PrepareTestProjects()
         {
-            var runtime = TestUtils.GetClrRuntimeComponents().First();
-            var runtimeHomeDir = TestUtils.GetRuntimeHomeDir((string)runtime[0], (string)runtime[1], (string)runtime[2]);
-            var dthTestProjectsSource = Path.Combine(TestUtils.GetMiscProjectsFolder(), "DthTestProjects");
+            var sdk = DnxSdkFunctionalTestBase.ClrDnxSdks.First();
+            var dthTestProjectsSource = Path.Combine(CommonTestUtils.TestUtils.GetMiscProjectsFolder(), "DthTestProjects");
 
             foreach (var testProject in Directory.GetDirectories(dthTestProjectsSource))
             {
-                TestUtils.CreateDisposableTestProject(
-                    runtimeHomeDir,
-                    _context.DirPath,
-                    testProject);
+                CreateDisposableTestProject((DnxSdk)sdk[0], _context.DirPath, testProject);
             }
         }
+
+        private static void CreateDisposableTestProject(DnxSdk sdk, string targetDir, string sourceDir)
```

Comment:
@davidfowl is this a blocking issue right now? I can work with @JunTaoLuo to find a better replacement later.